esphome/components/modbus_controller/select/modbus_select.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <utility>
+
+#include "esphome/components/modbus_controller/modbus_controller.h"
+#include "esphome/components/select/select.h"
+#include "esphome/core/component.h"
+
+namespace esphome {
+namespace modbus_controller {
+
+class ModbusSelect : public Component, public select::Select, public SensorItem {
+ public:
+  ModbusSelect(SensorValueType sensor_value_type, uint16_t start_address, uint8_t register_count, uint8_t skip_updates,
+               bool force_new_range, std::vector<int64_t> mapping)
+      : Component(), select::Select() {
```

Comment:
To fix the new clang-tidy check:
```suggestion
               bool force_new_range, std::vector<int64_t> mapping) {
```